spynnaker/pyNN/models/neuron/population_machine_vertex.py

```diff
@@ -627,3 +628,35 @@ def gen_on_machine(self):
     def read_generated_connection_holders(self, transceiver, placement):
         self._app_vertex.synapse_manager.read_generated_connection_holders(
             transceiver, placement)
+
+    @overrides(AbstractReadParametersBeforeSet.read_parameters_from_machine)
+    def read_parameters_from_machine(
+            self, transceiver, placement, vertex_slice):
+
+        # locate SDRAM address to where the neuron parameters are stored
+        neuron_region_sdram_address = self.neuron_region_sdram_address(
+            placement, transceiver)
+
+        # shift past the extra stuff before neuron parameters that we don't
+        # need to read
+        neuron_parameters_sdram_address = (
+            neuron_region_sdram_address +
+            self._app_vertex.tdma_sdram_size_in_bytes +
+            self._app_vertex.BYTES_TILL_START_OF_GLOBAL_PARAMETERS)
+
+        # get size of neuron params
+        size_of_region = self._app_vertex.get_sdram_usage_for_neuron_params(
+            vertex_slice)
+        size_of_region -= (
+            self._app_vertex.BYTES_TILL_START_OF_GLOBAL_PARAMETERS +
+            self._app_vertex.tdma_sdram_size_in_bytes)
```

Comment:
Similar to comment above; marking to keep track

Reply:
Resolved above

Reply:
refer to main view difference in impls.